### PR TITLE
Status Bar Updates based on Pixel Designs

### PR DESCRIPTION
**Pull request by yanfali**
Updates to  the UI based on design input from Pixel @wojciechgabrys 

Screen Shot 2019-11-22 at 12 07 16

### CODE REVIEW

**Comment by zekth** [comment]
Is it working properly in french / russian and so on ?

**Comment by yanfali** [comment]
> Is it working properly in french / russian and so on ?
> 
> You want a PR to fix the tests?

Yes, I broke compatibility with the old message.

File: src/components/StatusBar.vue [hunk 1]
```diff
@@ -94,4 +125,7 @@ export default {
 .bes-odd-job-count {
   color: yellow;
 }
+.bes-odd-job-count {
+  color: yellow;
+}
```
Comment (noroadsleft) [inline src/components/StatusBar.vue]: Bad copy-paste?

File: src/scss/style.scss [hunk 2]
```diff
@@ -1024,57 +765,6 @@ input[type='number'] {
...
-
-.bes-jobs {
-  justify-self: end;
-}
-.hidden {
-  position: absolute;
-  top: -1000px;
-  left: -1000px;
+.random-potato {
+  font-style: italic;
```
Comment (noroadsleft) [inline src/scss/style.scss]: I prefer the way the potato facts look without the italics, but I'll leave this up to you.

Comment (zekth) [reply src/scss/style.scss]: Jack mentionned it in the refactor of the configurator it was `quirky`. Do we still keep those? Also we can't put it in the veil of the compilation.

**Comment by zekth** [comment]
On large screen the spacing is a bit odd compared to your screenshot:
2019-11-23 10_36_17-QMK Configurator

May it be better to attach the status next to the qmk logo on the left?

Also regarding the labeling:
2019-11-23 10_41_04-QMK Configurator

Having `the server is` is a bit verbose and non revelant to have. Let's just put `API` and `current queue` with help labels on them. Would make i18n less impacting.

Last note, Merlin is a bit overlapped:
2019-11-23 10_43_11-QMK Configurator

It may be added next to `settings` ?

**Comment by yanfali** [comment]
@zekth that's an old version, run it locally

**Comment by yanfali** [comment]
Screen Shot 2019-11-23 at 11 26 26

**Comment by noroadsleft** [comment]
*In this episode of "Annoying Former Front-End Web Designer Makes Annoying Request..."*

```
SCENE: Indoors. Open-plan office. Mid evening.

noroadsleft: "I don't like the lack of contrast on the potato facts. They aren't clearly legible."

[yanfali: groans]

[noroadsleft: walks away]

[yanfali: sips coffee angrily to prevent himself from screaming in frustration]

END SCENE
```


```diff
diff --git a/src/scss/style.scss b/src/scss/style.scss
index ce03a9aba..4a942edf6 100644
--- a/src/scss/style.scss
+++ b/src/scss/style.scss
@@ -764,7 +764,3 @@ input[type='number'] {
   font: 14px/1.5 'Noto Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
   margin-left: 5px;
 }
-
-.random-potato {
-  color: #8f8f8f;
-}
diff --git a/src/scss/themes.scss b/src/scss/themes.scss
index 3b24410c9..f6f5b9992 100644
--- a/src/scss/themes.scss
+++ b/src/scss/themes.scss
@@ -340,6 +340,9 @@ html {
       }
     }
   }
+  .random-potato {
+    color: #2c3e50;
+  }
 }
 // Dark theme
 html[data-theme='dark'] {
@@ -457,4 +460,7 @@ html[data-theme='dark'] {
       }
     }
   }
+  .random-potato {
+    color: #eeeeee;
+  }
 }
```

**Comment by yanfali** [comment]
```diff
> diff --git a/src/scss/style.scss b/src/scss/style.scss
> index ce03a9aba..4a942edf6 100644
> --- a/src/scss/style.scss
> +++ b/src/scss/style.scss
> @@ -764,7 +764,3 @@ input[type='number'] {
>    font: 14px/1.5 'Noto Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
>    margin-left: 5px;
>  }
> -
> -.random-potato {
> -  color: #8f8f8f;
> -}
> diff --git a/src/scss/themes.scss b/src/scss/themes.scss
> index 3b24410c9..f6f5b9992 100644
> --- a/src/scss/themes.scss
> +++ b/src/scss/themes.scss
> @@ -340,6 +340,9 @@ html {
>        }
>      }
>    }
> +  .random-potato {
> +    color: #2c3e50;
> +  }
>  }
>  // Dark theme
>  html[data-theme='dark'] {
> @@ -457,4 +460,7 @@ html[data-theme='dark'] {
>        }
>      }
>    }
> +  .random-potato {
> +    color: #eeeeee;
> +  }
>  }
> ```

I was going for something less emphasized but this is fine.

**Comment by yanfali** [comment]
Screen Shot 2019-11-23 at 21 01 44

latest version

**Comment by yanfali** [comment]
Screen Shot 2019-11-23 at 21 03 38
This is what it shows when the queue is saturated

**Comment by noroadsleft** [comment]
> I was going for something less emphasized but this is fine.

We can go down some but I felt like the previous color was too far down.

**Comment by yanfali** [comment]
> > I was going for something less emphasized but this is fine.
> 
> We can go down some but I felt like the previous color was too far down.

I prefer to defer to you for styling, so please suggest values. I am mostly doing structural work

**Comment by yanfali** [comment]
@noroadsleft while you are in there, can you center the potato fact vertically a bit? Thanks

**Comment by noroadsleft** [comment]
> I prefer to defer to you for styling, so please suggest values. I am mostly doing structural work

Will fix in a separate PR. Carry on.

**Comment by yanfali** [comment]
@noroadsleft should we merge this?

File: src/scss/themes.scss [hunk 3]
```diff
@@ -157,7 +157,7 @@ html {
       background: #eee;
     }
     .backend-status {
-      background: #222;
+      background: #000;
```
Comment (jackhumbert) [inline src/scss/themes.scss]:
```suggestion
      background: #272822;
```

File: src/scss/status-bar.scss [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  grid-template: 1fr / [qmk-logo] auto [qmk-app-name] auto [api-version];
+}
+
+.qmk-logo {
+  grid-column: qmk-logo;
+  background-image: url('../assets/qmk_icon_36.png');
+  width: 36px;
+  height: 36px;
+  transform: scale(0.8);
+}
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  margin-top: 2px;
}
```

File: src/scss/status-bar.scss [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+  display: grid;
+  grid-template: 1fr / [qmk-logo] auto [qmk-app-name] auto [api-version];
+}
+
+.qmk-logo {
+  grid-column: qmk-logo;
+  background-image: url('../assets/qmk_icon_36.png');
+  width: 36px;
+  height: 36px;
+  transform: scale(0.8);
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  transform: scale(0.777777);
```

**Comment by jackhumbert** [comment]
I think a full white logo will look better here: 

qmk-36x36

preview

File: src/scss/status-bar.scss [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+}
+.qmk-app-name {
+  grid-column: qmk-app-name;
+  color: #fff;
+  background-repeat: no-repeat;
+  padding-left: 10px;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  padding-left: 3px;
```

File: src/scss/status-bar.scss [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+  overflow: hidden;
+  white-space: nowrap;
+  text-overflow: ellipsis;
+}
+
+.bes-version {
+  grid-column: api-version;
+  margin-left: 10px;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  margin-left: 6px;
```

File: src/scss/status-bar.scss [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+  justify-self: center;
+  font-size: 12px;
+}
+
+.bes-status {
+  grid-column: server-status;
+  display: grid;
+  grid-template: 1fr / auto auto auto;
+  width: auto;
+  background-color: #243a2c;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  background-color: #24402e;
```

File: src/scss/status-bar.scss [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+  max-width: 5rem;
+  overflow: hidden;
+  border-right: 1px solid #000;
+  width: 7rem;
+  text-align: center;
+}
+
+.bes-status-right {
+  padding: 0 8px;
+  min-width: 7rem;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  min-width: 5rem;
```

File: src/scss/status-bar.scss [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+  }
+  width: 1rem;
+}
+
+.bes-status-center {
+  padding: 0 4px;
+  max-width: 5rem;
+  overflow: hidden;
+  border-right: 1px solid #000;
+  width: 7rem;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  width: 5rem;
```

File: src/scss/status-bar.scss [hunk 11]
```diff
@@ -0,0 +1,111 @@
...
+  }
+  width: 1rem;
+}
+
+.bes-status-center {
+  padding: 0 4px;
+  max-width: 5rem;
+  overflow: hidden;
+  border-right: 1px solid #000;
+  width: 5rem;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
```

File: src/scss/status-bar.scss [hunk 12]
```diff
@@ -0,0 +1,111 @@
...
+    height: 7px;
+    margin-right: 0px;
+    border-radius: 100%;
+  }
+  width: 1rem;
+}
+
+.bes-status-center {
+  padding: 0 4px;
+  max-width: 5rem;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
```

File: src/scss/status-bar.scss [hunk 13]
```diff
@@ -0,0 +1,111 @@
...
+    width: 7px;
+    height: 7px;
+    margin-right: 0px;
+    border-radius: 100%;
+  }
+  width: 1rem;
+}
+
+.bes-status-center {
+  padding: 0 4px;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  padding: 0 30px;
```

File: src/scss/status-bar.scss [hunk 14]
```diff
@@ -0,0 +1,111 @@
...
+  max-width: 5rem;
+  overflow: hidden;
+  border-right: 1px solid #000;
+  width: 5rem;
+  text-align: center;
+}
+
+.bes-status-right {
+  padding: 0 8px;
+  min-width: 5rem;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
```

File: src/scss/status-bar.scss [hunk 15]
```diff
@@ -0,0 +1,111 @@
...
+  padding: 0 4px;
+  max-width: 5rem;
+  overflow: hidden;
+  border-right: 1px solid #000;
+  width: 5rem;
+  text-align: center;
+}
+
+.bes-status-right {
+  padding: 0 8px;
```
Comment (jackhumbert) [inline src/scss/status-bar.scss]:
```suggestion
  padding: 0 30px;
```

**Comment by yanfali** [comment]
@noroadsleft should we just merge this? I can live with the size changing behavior for now

**Comment by noroadsleft** [comment]
> @noroadsleft should we just merge this? I can live with the size changing behavior for now

Yeah, it's fine. I too would like to find a solution for the size changing though, because it bothers me too.

I'll probably do a PR with some CSS polishing soon.

**Pull request merged**